Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -316,6 +320,20 @@ void Mixer::PushWiimoteSpeakerSamples(const short* samples, unsigned int num_sam
   }
 }
 
+void Mixer::PushSkylanderPortalSamples(const short* samples, unsigned int num_samples)
```

Comment:
`num_samples` is effectively hardcoded to 32 here thanks to the `std::array`. That's fine, but you should probably remove the parameter.